tests/e2e/vsphere_volume_expansion.go

```diff
@@ -659,7 +659,7 @@ var _ = ginkgo.Describe("Volume Expansion Test", func() {
 		gomega.Expect(pvclaim).NotTo(gomega.BeNil())
 
 		ginkgo.By("File system resize should not succeed Since SPS service is down. Expect an error")
-		expectedErrMsg := "ServerFaultCode: CnsFault"
+		expectedErrMsg := "failed to expand volume"
```

Comment:
The test might fail if there is a change in expected error message.

Reply:
Error message has changed
and "failed to expand volume" looks more valid